src/extensions/askUserForConsent.ts

```diff
@@ -64,6 +65,33 @@ export async function displayRoles(
   utils.logLabeledBullet(logPrefix, message);
 }
 
+/**
+ * Displays APIs that will be enabled for the project and corresponding descriptions.
+ * @param extensionName name of extension to install/update
+ * @param projectId ID of user's project
+ * @param apis APIs that require user approval
+ */
+export async function displayApis(
```

Comment:
Doesn't seem like there is any async calls in here?

Reply:
Done.